sql/core/src/test/scala/org/apache/spark/sql/connector/V2CommandsCaseSensitivitySuite.scala

```diff
@@ -186,7 +191,7 @@ class V2CommandsCaseSensitivitySuite extends SharedSparkSession with AnalysisTes
   test("AlterTable: drop column resolution") {
     Seq(Array("ID"), Array("point", "X"), Array("POINT", "X"), Array("POINT", "x")).foreach { ref =>
       alterTableTest(
-        TableChange.deleteColumn(ref),
+        AlterTableDropColumns(table, Seq(UnresolvedFieldName(ref))),
```

Comment:
This one was (and is) working fine because handling `DeleteColumn` in `Analyzer` will not be removed until "replace" command is migrated.